Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebViewModel.swift

```diff
@@ -6,8 +6,7 @@ class PayPalWebViewModel: ObservableObject, PayPalWebCheckoutDelegate {
 
     @Published var state: CurrentState = .idle
     @Published var intent: Intent = .authorize
-    @Published var createOrderResult: Order?
-    @Published var transactionResult: Order?
+    @Published var order: Order?
```

Comment:
We can get rid of the duplicate `Order` above now that we have a singular result view 🎉

Reply:
ux: on Android we do display multiple snapshots of the `Order` at different points in time. Not sure how it's designed on iOS, but adding some context about why these two values are separate.

Reply:
We were mainly using this as a psudo state on iOS to display slightly different results. Since we have 1 result that is appended to as we get more data, we don't need 2 properties.

Reply:
Hey @jaxdesmarais , it looks really smooth! I am wondering about replacing result view from create with authorize/capture result view. I liked having history of the states, I think that's why we retained the different order snapshots in the first place, to compare the results to make sure that things match up. One thing that is kind of unexpected in my opinion is that the result view after order creation appears after the toggle for the funding source.

Reply:
Just getting back to these comments from being on PTO. 😄 

> I am wondering about replacing result view from create with authorize/capture result view. I liked having history of the states, I think that's why we retained the different order snapshots in the first place, to compare the results to make sure that things match up.

We can use breakpoints in these cases but should not busy up our demo app with different views containing the same data type. We could also add a quick print statement of the states if needed in each method that returns an order.

> Another thing that is kind of unexpected for me is that the result view after order creation appears after the toggle for the funding source.

This made the most sense to keep the result view pinned in the same general place throughout the flow without it jumping around as we added subsequent views. Otherwise, we'd require unnecessary logic to move this view around throughout the flow. I had tested this initially and it was much cleaner to keep the result view relatively in 1 place.